client/state/reader/teams/reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External Dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_TEAMS_REQUEST,
+	READER_TEAMS_REQUEST_FAILURE,
+	READER_TEAMS_REQUEST_SUCCESS,
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const items = createReducer( [], {
+	[ READER_TEAMS_REQUEST_SUCCESS ]: ( state, action ) => [ ...state, ...action.teams ],
```

Comment:
Is there no dependence here on site or site id? Are all of the teams supposed to join into one big list? (this may be fine, but it's odd-looking to me when I don't know any better)

is the list here the best way to represent the teams? do teams have an `id`? if so, should we look into the [keyedReducer](https://github.com/Automattic/wp-calypso/blob/c059c5a4dd07076428ccabfcd2db7567c1fe1d01/client/state/utils.js#L77) helper?